### ISSUE TITLE
Space added in "output" with auto-lineup declarations

### ISSUE THREAD

**Issue by sjalloq** [veripool/verilog-mode#1760]
Hi,

I'm using verilog-mode with Xemacs in batch mode to run auto's and indentation.  The following testcase highlights two issues with the indentation.

1. The `output` keyword is split with a space
2. The top module instance indentation is broken within a for loop.

I've checked this with the latest version of verilog-mode.

```systemverilog
module test (
input logic a,
output logic b
);

for (genvar i=0; i<2; i++) begin : g_slice

submod
u_sub_0 #(
.RESET_POL (RESET_POL)
)(
.clk     (clk),
.reset_n (reset_n),
.d       (d[i]),
.q       (q_0[i])
);

submod
u_sub_1 #(
.RESET_POL (RESET_POL)
)(
.clk     (clk),
.reset_n (reset_n),
.d       (q_0[i]),
.q       (q[i])
);

end

endmodule
```

Output after running verilog-batch-indent:

```systemverilog
module test (
             input logic a,
             o utput logic b
             );

  for (genvar i=0; i<2; i++) begin : g_slice

    submod
                u_sub_0 #(
                          .RESET_POL (RESET_POL)
                          )(
                            .clk     (clk),
                            .reset_n (reset_n),
                            .d       (d[i]),
                            .q       (q_0[i])
                            );

    submod
      u_sub_1 #(
                .RESET_POL (RESET_POL)
                )(
                  .clk     (clk),
                  .reset_n (reset_n),
                  .d       (q_0[i]),
                  .q       (q[i])
                  );

  end

endmodule
```

**Comment by wsnyder** [comment]
This is not at all what I get, I'm using verilog-mode-version "2022-01-02-e326b90-vpo" and xemacs 21.4 patch 24.

However, I would caution you that xemacs hasn't been upgrading for over 20 years, so you should not be using it, move to Gnu Emacs.

Closing as assuming upgrading will fix it, if not feel free to comment/reopen.

**Comment by sjalloq** [comment]
Sorry, I said Xemacs when I mean Emacs as I had just compiled the latest version of verilog-mode and had it in my mind.

I'm using GNU Emacs v27.2.  I've attached a full testcase for you.  You'll just have to edit the path to emacs in `util/verilog_indent`.  Run the example using `util/verilog_indent test.sv`.

[testcase.tgz.zip](https://github.com/veripool/verilog-mode/files/7942536/testcase.tgz.zip)
